### PR TITLE
Feature/pm 391 sm to unknown button

### PR DESCRIPTION
**Pull request by Joris-JW**
Closes PM-391
(Also use the feature/PM-391-set-state-machine-to-unknown in the march repo)

## Description
Added a button with a callback that sends the UNKNOWN tag as GaitInstruction.msg

## Changes
* new button to set the state machine to unknown
* callback of the button

### CODE REVIEW

**Review by RutgerVanBeek — APPROVED** [review]
Wait for merging until the one in the March repo is merged.

**Review by Roelemans — APPROVED** [review]

**Pull request merged**
